src/cuga/backend/skills/registry.py

```
@@ -0,0 +1,55 @@
+            if pip_pkgs:
+                setup_lines.append(f"await run_command('pip3 install --quiet {' '.join(pip_pkgs)}')")
+            if npm_pkgs:
+                # Install locally in the working dir so require() resolves correctly
+                setup_lines.append(
+                    f"await run_command('cd /tmp/cuga_workspace && npm install {' '.join(npm_pkgs)}')"
+                )
+            setup_script = "\n".join(setup_lines)
+            parts.append(f"STEP 1 — SETUP (run first, in a ```python``` code block):\n{setup_script}")
+            parts.append("")
+
+        skill_dir = f"/tmp/cuga_workspace/skills/{entry.name}"
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/cuga_workspace/skills/"

(S108)